WeakAuras/WeakAuras.lua

```diff
@@ -4055,7 +4055,9 @@ function Private.FixGroupChildrenOrderForGroup(data)
   if data.parent == nil then
     for child in Private.TraverseAll(data) do
       SetFrameLevel(child.id, frameLevel);
-      frameLevel = frameLevel + 4;
+      if data.regionType ~= "dynamicgroup" then
```

Comment:
Instead of checking for each child, either check once before iterating or make the offset a parameter of the function and call it with the correct offset from Group/DynamicGroup

Reply:
done